components/suggestion/switch_channel_provider.jsx

```diff
@@ -7,7 +7,6 @@ import {connect} from 'react-redux';
 import classNames from 'classnames';
```

Comment:
@hmhealey `classNames` import is there, but still ESlint is failing with this error

Reply:
You've actually got a casing issue. The import is `classNames` but you're using `classnames` in your new code

Reply:
🤦‍♂️, thanks man